rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxView.java

```diff
@@ -147,6 +148,20 @@
   }
 
   /**
+   * Create an observable of {@link ViewTreeObserverDrawEvent} for draws on {@code view}.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   * <p>
+   * <em>Warning:</em> The created observable uses {@link ViewTreeObserver#addOnDrawListener} to observe
+   * draws. Only one observable can be used for a view at a time.
```

Comment:
ViewTreeObserver allows more than one OnDrawListener to be registered therefore this doc line should be changed according to https://github.com/JakeWharton/RxBinding/issues/75.